website/hw/hw8.rst

```diff
@@ -651,15 +781,14 @@ Now, from your pipeline’s page, do the following:
 Now, you’ll create a pull request:
```

Comment:
The "Enable Review Apps" button appeared on the side before the checkboxes appeared on the main page

Reply:
I'm not sure what this refers to. What checkboxes do you mean?

Reply:
Nevermind, this was due to my being on the settings page.